NewForce-Cohort-4/tabloid-full-stack-team-fuschia
# Pp view all post

This PR is for Ticket #3

Ticket #3
> As a reader, I would like to see a list of all the Posts so that I can choose a post that seems interesting to read.
> 
> **Given** the user is in the Tabloid application
> **When** they select the `Posts` menu option
> **Then** they should be directed to the Posts list page
> **And** each post in the list should display the title, author and category
> **And** the list should ONLY contain approved Posts
> **And** the list should ONLY contain Posts with a publication date that is in the past
> **And** the list should be in order of publication date with the most recent on top




Type of change
Please delete options that are not relevant.

X New feature (non-breaking change which adds functionality)

Testing Instructions
pull down the pp-view-all-post branch and start the application
Login and navigate to the Posts option in the nav bar
Click on Post
You should be navigated to a list of all the Post


Checklist:
 My code follows the style guidelines of this project
 I have performed a self-review of my own code
 I have commented my code, particularly in hard-to-understand areas
 My changes generate no new warnings or errors
 I have added test instructions that prove my fix is effective or that my feature works
